docker/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM gradle:7.1-jdk16 AS build
+FROM gradle:7.2-jdk16 AS build
```

I think we should keep them in sync now since Renovate will update both
```suggestion
FROM gradle:7.4-jdk17 AS build
```